package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "p2p-graph",
   "description": "Real-time P2P network visualization with D3",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
Bumping the version shouldn't be part of a PR. What if we get another PR which also bumps the version and that one gets merged first? Then this version change is incorrect. Version changing should happen *after* a PR is merged, using `npm version patch`, `npm version minor`, etc.